src/dev/src/app/datagrid/kitchen-sink/kitchen-sink-data.ts

```diff
@@ -8,7 +8,7 @@ export class DatagridKitchenSinkData {
     {
       id: 3379,
       name: 'Rhona',
-      creation: '2014-04-11T03:02:57.974Z',
+      creation: '2014-04-11T20:02:57.974Z',
```

Comment:
Moving the time to later in the day prevented different days on different computers (my local vs applitools). Some for the other changes like this below.